src/current/_includes/v22.2/misc/assume-role-description.md

```diff
@@ -0,0 +1 @@
+Pass the [ARN](https://docs.aws.amazon.com/general/latest/gr/aws-arns-and-namespaces.html) of the role to assume. Use in combination with `AUTH=implicit` or `specified`. <span class="version-tag">New in v22.2.6:</span> `external_id`: Use as a value to `ASSUME_ROLE` to specify the [external ID](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create_for-user_externalid.html) for third-party access to your S3 bucket.
```

Comment:
I have added a reference to the Cloud Storage Auth page in the backup.md as that did not have any link for `assume_role`. This avoids adding logic to the include file to self-referencing the page, because this include is also used on the cloud storage auth page itself.